apps/deploy_monitoring/src/deploy_with_compose.rs

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) SimpleStaking and Tezedge Contributors
+// SPDX-License-Identifier: MIT
+#![forbid(unsafe_code)]
+
+use std::path::PathBuf;
+use std::process::{Command, Output};
+
+use slog::{info, Logger};
+use tokio::time::{sleep, Duration};
+
+use crate::image::{DeployMonitoringContainer, Explorer, Sandbox, TezedgeDebugger};
+use crate::node::{OcamlNode, TezedgeNode, OCAML_PORT, TEZEDGE_PORT};
+
+pub const DEBUGGER_PORT: u16 = 17732;
+
+pub async fn launch_stack(compose_file_path: &PathBuf, log: &Logger) {
+    info!(log, "Tezedge explorer is starting");
+    start_with_compose(compose_file_path, Explorer::NAME, "explorer");
+    wait_for_start(&format!("http://localhost")).await;
```

Comment:
@adonagy 
clippy, just change `(&format!("http://localhost")` to `"http://localhost"`,
explorer does not have port?
I guess it take 80 as default, so I would prefer to use a constant here with 80

Reply:
Exactly, the explorer uses 80, I'll add it as a constant